queries/typescript/highlights.scm

```diff
@@ -11,6 +11,7 @@
   "namespace"
   "override"
   "satisfies"
+  "module"
```

Comment:
i guess that's ok here. Some `module` keywords from other languages might be  `@include`

Reply:
I agree, but for TypeScript, `module` is synonymous with the `namespace` keyword, with a little difference when used with the `declare` keyword.